### ISSUE TITLE
useLayoutEffect in SSR context

### ISSUE THREAD

**Issue by shehi** [OlegWock/react-local-toast#7]
```
Warning: useLayoutEffect does nothing on the server, because its effect cannot be encoded into the server renderer's output format. This will lead to a mismatch between the initial, non-hydrated UI and the intended UI. To avoid this, useLayoutEffect should only be used in components that render exclusively on the client. See https://reactjs.org/link/uselayouteffect-ssr for common fixes.
    at /var/www/node_modules/react-local-toast/dist/react-local-toast.cjs.development.js:68:27
    at /var/www/node_modules/react-local-toast/dist/react-local-toast.cjs.development.js:271:25
```

Would it be viable to switch to `useEffect()`? I use this package in a NextJs project and SSR pages cause this error.

Additionally, this error cascades into prop-mismatch errors (between server-rendered and hydrated pages) only where this package is used.

**Comment by OlegWock** [comment]
Hello @shehi. This should be fixed in `1.1.4`, could you give it a try?

**Comment by shehi** [comment]
Fixed, thanks! :D
